common/src/KokkosFFT_transpose.hpp

```diff
@@ -13,62 +13,60 @@
 namespace KokkosFFT {
 namespace Impl {
 template <typename ViewType, std::size_t DIM>
-auto get_map_axes(const ViewType& view, axis_type<DIM> _axes) {
-  KOKKOSFFT_THROW_IF(!KokkosFFT::Impl::are_valid_axes(view, _axes),
+auto get_map_axes(const ViewType& view, axis_type<DIM> axes) {
+  KOKKOSFFT_THROW_IF(!KokkosFFT::Impl::are_valid_axes(view, axes),
                      "get_map_axes: input axes are not valid for the view");
 
   // Convert the input axes to be in the range of [0, rank-1]
-  std::vector<int> axes;
+  axis_type<DIM> non_negative_axes = {};
   for (std::size_t i = 0; i < DIM; i++) {
-    int axis = KokkosFFT::Impl::convert_negative_axis(view, _axes.at(i));
-    axes.push_back(axis);
+    int axis = KokkosFFT::Impl::convert_negative_axis(view, axes.at(i));
+    non_negative_axes[i] = axis;
```

Comment:
```suggestion
    non_negative_axes[i] = KokkosFFT::Impl::convert_negative_axis(view, axes.at(i));
```

Seems ok to do it at once ?

Reply:
Certainly, I will do that